src/current/v24.1/ui-overview.md

```diff
@@ -146,6 +146,11 @@ The DB Console stores temporary data in a time-series database in order to gener
 
 By default, the DB Console shares anonymous usage details with Cockroach Labs. For information about the details shared and how to opt-out of reporting, see [Diagnostics Reporting]({% link {{ page.version.version }}/diagnostics-reporting.md %}).
 
+## License expiration alert
```

Comment:
Can we call this a "message" instead of alert? I worry that "alert" will imply to customers that they can get some sort of notification sent.

Reply:
Thanks for making that distinction! I replaced "alert" with "message".